src/routes/our-music/index.svelte

```diff
@@ -19,28 +146,87 @@
             align-items: center;
             flex-wrap: wrap;
         }
+        .instrument-container {
+            display: flex;
+            border: 3px;
+            padding: 10px;
+            margin: 5px;
+            align-content: space-between;
+            justify-content: center;
+            align-items: flex-start;
+            flex-wrap: wrap;
+        }
         .special-container {
             display: flex;
             position: relative;
             border: 3px;
-            padding: 15px;
-            margin: 20px;
+            padding: 8px;
+            margin: 7px;
             align-content: space-between;
             justify-content: center;
             align-items: center;
             flex-wrap: wrap;
         }
+        .box {
+            flex: 1;
+            margin: 18px;
+        }
+        .box-small-margin {
+            flex: 1;
+            margin: 5px;
+        }
     }
-    .box {
-        flex: 1;
-        flex-basis: 20%;
-        margin: 20px;
+    .box-profile-picture {
+        flex: 2;
+        margin: 5px;
     }
-    .box-small-margin {
-        flex: 1;
-        flex-basis: 20%;
+    .box-audio-player {
+        flex: 5;
         margin: 5px;
+        justify-content: flex-start;
+        min-width: 0;
+        overflow: hidden;
+        text-overflow: ellipsis;
+    }
+    /*.song-container-header {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-between;
+        align-items: stretch;
+        align-content: center;
+        font-size: 16px;
+        flex-wrap: nowrap;
     }
+    .song-container {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-between;
+        align-items: stretch;
+        align-content: center;
+        font-size: 14px;
+        flex-wrap: nowrap;
+        border-bottom-style: solid;
+        border-width: 1px;
+    }
+    .song-box-small {
+        flex: 1;
+        padding: 6px;
+        min-width: 0;
+        overflow: hidden;
+        white-space: nowrap;
+    }
+    .song-box-med {
```

Comment:
It's not a good idea to keep the commented code in our commits. Most of the time, we'll forget what are these commented codes doing. We'll just remove it for now and in the worst case if we still need it, we can still refer to our github history.